cime_config/config_pes.xml

```diff
@@ -184,7 +184,7 @@
           <ntasks_rof>128</ntasks_rof>
           <ntasks_cpl>128</ntasks_cpl>
           <ntasks_ice>128</ntasks_ice>
-          <ntasks_ocn>384</ntasks_ocn>
+          <ntasks_ocn>1024</ntasks_ocn>
```

Comment:
I'm a little surprised that we give MOM more tasks (1024) when running with WW3 than we do with SWAV (896, see line 121). Is this intentional?

Reply:
Agreed. I just set NTASKS_OCN to 896 instead of 1024.